src/components/ErrorDialog.tsx

```diff
@@ -18,6 +18,7 @@ export const ErrorDialog = ({
     if (onClose) {
       onClose();
     }
+    document.querySelector<HTMLElement>(".excalidraw")!.focus();
```

Comment:
if we're not gonna be passing down the `Excalidraw.focusContainer()` method, we should at least factor this into util, and also narrow down the selector to guarantee we don't focus something else, like a modal.